scss/partials/_general.scss

```diff
@@ -32,6 +32,10 @@ input[type=submit] {
 .container {
 	width: 90%;
 	margin: 0 auto;
+
+	h4{
```

Comment:
The file is using space between the selector name and the block initializer, you should follow the pattern.
`h4 {`